Removing unused/obsolete libraries from lib-src

lllucius opened the pull request:
Libs removed:

id3lib
libresample
libsamplerate
portburn
taglib

This effectively makes libsoxr THE resampling lib for Audacity.

In addition, changes were made to include it in the minsrc tarball.

JamesCrook commented:
This will not automatically merge.

might portburn be useful in the future?

Closing this pull request, and suggesting that instead you just remove things you are fairly sure we won't need.  We can always add back (more up to date versions) later if we want them.

lllucius commented:
Actually, I find it unlikely that portburn would be useful in the future.  Dominic started it on OSX.  I updated and added the Windows stuff.  But, it's been years and I doubt that it will ever see the light of day (or if it even still works).

Even so, I'll removed the other libs and leave portburn.  The only one without an upstream version is libresample and we can always pull it from history if needed.
